File: examples/javascript/orphaned-module.js
// This module is completely unused and should be detected as dead

function orphanedFunction() {
    // This function is in an orphaned file
    return "This file is never imported";
}

class OrphanedClass {
    constructor() {
        this.value = 42;
    }
    
    method() {
        return this.value * 2;
    }
}

module.exports = {
    orphanedFunction,
    OrphanedClass
};

File: examples/python/orphaned_module.py
"""
This module is completely unused and should be detected as dead.
"""

def orphaned_function():
    """This function is in an orphaned file."""
    return "This file is never imported"

class OrphanedClass:
    """This class is in an orphaned file."""
    
    def __init__(self):
        self.value = 42
    
    def method(self):
        return self.value * 2